_posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Trino Summit 2022 recap: The state of Trino"
+author: "Martin Traverso, Dain Sundstrom, David Phillips, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/keynote-header.png
+---
+
+Welcome to the first in a series of posts recapping everything interesting that
+went down at the 2022 Trino Summit! There is a post being published for each
```

Comment:
I dont think we should explain what we are doing .. just remove the first two sentences and replace it with a link to the post as part of the next sentence 


```
To kick off [Trino Summit 2022](link), we heard
```